entry/src/main/js/default/pages/index/index.hml

```diff
@@ -0,0 +1,98 @@
+<element src="../../../../../../../smartwash/js/index.hml" name="test"></element>
+<div class="container">
+    <div class="full">
+        <div class="timer">
+            <image class="level_bg" src="/common/images/level_bg.png"></image>
+            <image class="Timmer_bg" src="/common/images/Timer_bg.png"></image>
```

Comment:
Include the functionality like for example for wash type, user clicks on spin shall update UI accordingly.